includes/data-stores/class-wc-admin-reports-categories-data-store.php

```diff
@@ -59,6 +59,16 @@ class WC_Admin_Reports_Categories_Data_Store extends WC_Admin_Reports_Data_Store
 		'products_count' => 'COUNT(DISTINCT product_id) as products_count',
 	);
 
+	/**
+	 * Constructor
+	 */
+	public function __construct() {
+		global $wpdb;
+		$table_name = $wpdb->prefix . self::TABLE_NAME;
+		// Avoid ambigious column order_id in SQL query.
+		$this->report_columns['orders_count'] = str_replace( 'order_id', $table_name . '.order_id', $this->report_columns['orders_count'] );
```

Comment:
It would be simpler, but I think it would cause more confusion when debugging and means we are defining this value twice instead of filtering it.  

For example, if we redefine this variable entirely and someone changes the value in `$report_columns` instead of the construct method, no errors occur, but the code does not work as intended.

On the other hand, if we filter the value and the user changes the `$report_columns` value from:
`'orders_count'   => 'COUNT(DISTINCT order_id) as orders_count'` to
`'orders_count'   => 'COUNT(order_id) as orders_count'`
then the function continues to work as expected.  And if we filter a wrong value like `'orders_count'   => 'COUNT(DISTINCT order_id) as order_id_alias'`, we at least get an error message.

Neither one of these are very ideal solutions though, so I've opened another issue to address how we deal with ambiguous columns [here](https://github.com/woocommerce/wc-admin/issues/1308).  If it's okay with you, we can deal with that in another PR.